nipype/pipeline/engine/tests/test_engine.py

```diff
@@ -626,6 +626,33 @@ def func1(in1):
     assert not error_raised
 
 
+def test_parameterize_dirs_false(tmpdir):
+    from ....interfaces.utility import IdentityInterface
+    from ....testing import example_data
+
+    input_file = example_data('fsl_motion_outliers_fd.txt')
+
+    n1 = pe.Node(EngineTestInterface(), name='Node1')
+    n1.iterables = ('input_file', (input_file, input_file))
+    n1.interface.inputs.input1 = 1
+
+    n2 = pe.Node(IdentityInterface(fields='in1'), name='Node2')
+
+    wf = pe.Workflow(name='Test')
+    wf.base_dir = str(tmpdir)
+    wf.config['execution']['parameterize_dirs'] = False
+    wf.connect([(n1, n2, [('output1', 'in1')])])
+
+    error_raised = False
+    try:
+        wf.run()
+    except Exception as ex:
```

Comment:
Good question. I was looking at the other tests, and they all seemed to catch generic exceptions. It dawned on me that if this is a test of a configuration specific execution pathway, that maybe it was better to catch a general exception in case there was another reason for failure and the assertion would still work rather than bailing out.